uptane-standard.md

```diff
@@ -868,11 +882,14 @@ If the ECU performing the verification is the primary ECU, it SHOULD also ensure
 3. Check that it has been signed by the threshold of keys specified in the relevant metadata file (Checks for an arbitrary software attack):
     1. If checking top-level targets metadata, the threshold of keys is specified in the Root metadata.
     2. If checking delegated targets metadata, the threshold of keys is specified in the targets metadata file that delegated authority to this role.
-4. Check that the version number of the previous Targets metadata file, if any, is less than or equal to the version number of this Targets metadata file. (Checks for a rollback attack.)
-5. If this Targets metadata file indicates that the Timeserver key should be rotated, then reset the clock used to determine the expiration of metadata to a minimal value (e.g. zero, or any time that is guaranteed to not be in the future based on other evidence). It will be updated in the next cycle.
-6. Check that the current (or latest securely attested) time is lower than the expiration timestamp in this Targets metadata file. (Checks for a freeze attack.)
-7. If checking targets metadata from the Director repository, verify that there are no delegations.
-8. If checking targets metadata from the Director repository, check that no ECU identifier is represented more than once.
+4. Validate the signatures of the Targets metadata.
+	1. If checking top-level targets metadata signatures, the public keys are specified in the Root metadata.
+	2. If checking delegated targets metadata signatures, the public keys are specified in the targets metadata file that delegated authority to this role.
```

Comment:
I agree with @tkfu that this is redundant with the preceding step.

Reply:
Fair.
I removed and added clarity to previous step.